Pull changes on Sonarqube files

Mahbub-Hasan-Talukder opened the pull request.

Mahbub-Hasan-Talukder committed:
Revert "Fix/Sonar-file: Fix paths on sonarcloud.yml file and sonar-project.properties file"

This reverts commit 35e8f1140c21ded362fdd18b2984cd8f646774aa.

Warhammer4000 commented:
![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/warhammer4000/pr-checks/3d97516a-9b96-4e73-959e-720f089a753e?refs=e826773e032bd96b37a86d7b96abb63627448187&source=prChecksComment)

Mahbub-Hasan-Talukder committed:
Fix/Sonar-file: Fix paths on sonarcloud.yml file and sonar-project.properties file

Fix/Sonar-file: Fix paths on sonarcloud.yml file and sonar-project.properties file
